salt/search/config/etc-elasticsearch-elasticsearch.yml

```diff
@@ -0,0 +1,6 @@
+# Adapted from: https://github.com/elifesciences/builder-base-formula/blob/master/elife/config/etc-elasticsearch-elasticsearch.yml
+# Elasticsearch configuration for {{ grains['fqdn'] }}
+
+node.name: "{{ grains['fqdn'] }}"
```

Comment:
@lsh-0 should we use something more modern here like `{{ salt['elife.cfg']('project.stackname') }}`?